deployment/staging.yaml

```diff
@@ -31,8 +31,7 @@ steps:
   entrypoint: 'bash'
   args: [ '-c', "rm /workspace/service_account.json" ]
 - name: 'gcr.io/oss-vdb/deployment'
-  args: ['bash', '-ex', 'deploy.sh', 'oss-vdb', 'app.yaml', '--no-promote']
-  dir: gcp/appengine
+  args: ['bash', '-ex', 'gcp/appengine/deploy.sh', 'oss-vdb', 'deployment/oss-vdb/gae', 'app.yaml', '--no-promote']
```

Comment:
I think the project ID here should be `oss-vdb-staging` based on the filename?

Reply:
This is the staging instance within the prod GCP project, so this is intentional.